content/Developers.md

```diff
@@ -37,7 +37,7 @@ Eclipse VOLTTRON™ is an open source, secure, scalable, and distributed platfor
  -->
 
 **Eclipse VOLTTRON™ Resources**  
-The Eclipse VOLTTRON development team supports the wider user community with numerous resources in an effort to maximize transparency and simplify working with the platform. The project is hosted on GitHub for source control, issue tracking, and documentation. Visit Read the Docs <https://eclipse-volttron.readthedoc.io/> for more information.
+The Eclipse VOLTTRON development team supports the wider user community with numerous resources in an effort to maximize transparency and simplify working with the platform. The project is hosted on GitHub for source control, issue tracking, and documentation. Visit Read the Docs <https://eclipse-volttron.readthedocs.io/en/latest/> for more information.
```

Comment:
This should be what the old one was, it just needed the s in docs.  The en/latest is there for us to modify and send them to a different place based upon language preferences and default version (which we can change).